pages/index.js

```diff
@@ -1,3 +1,34 @@
+import styled from "styled-components";
+import movieData from "../db.json";
+
 export default function Home() {
-  return <h1>Hello Flamingos! 🦩</h1>;
+  return (
+    <main>
+      <h1>Your Movies</h1>
+      <br />
+      {movieData &&
+        movieData.map((movie) => {
+          return (
+            <StyledUl key={movie.id}>
+              <FancyLi>{movie.original_title}</FancyLi>
+              <StyledLi>Veröffentlichungsdatum: {movie.release_date}</StyledLi>
+              <StyledLi>Laufzeit: {movie.runtime} Min.</StyledLi>
+            </StyledUl>
```

Comment:
Currently you are rendering for each movie an `ul` element. I would recommend you to render all movies in a list and each movie would be basically an `li` element (💡 Maybe have a look again at the quiz-app).
You could also use a `dl` to display the information for each movie.